docs/src/examples/FlexBoxExample/FlexBoxExampleFirst.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { FlexBox } from '../../../../src/components/';
+import * as styles from '../../styles/components-page.scss';
+import { FlexAlign, FlexDirection, FlexJustify } from '../../../../src/components/FlexBox/FlexProps';
+
+export interface IProps{
+}
+
+export interface IState {
+}
+
+class FlexBoxExampleFirst extends React.Component<IProps> {
+  constructor(props: IProps) {
+    super(props);
+    this.state = {
+    };
+  }
+
+  valueUpdater(field: any) {
+    return (value: any) => this.setState({ [field]: value });
+  }
+
+  render() {
+    return (
+                <div className={styles.example}>
```

Comment:
everything inside of a .ts or .tsx file should only have 2 spaces of indentation.